src/theme/components/breadcrumb.ts

```diff
@@ -74,6 +74,35 @@ const whatsOn = defineStyle({
     bg: "dark.section.whats-on.secondary",
   },
 });
+const digitalCollections = defineStyle({
```

Comment:
Edwin already mentioned this, but take a look at how `customFocusColor` is used for the `blogs` variant.  The light background color necessitates a dark forcus ring.